src/pheval/runners/runner.py

```diff
@@ -3,7 +3,10 @@
 from dataclasses import dataclass
 from pathlib import Path
 
+from google.protobuf.timestamp_pb2 import Timestamp
```

Comment:
Thats an odd dependency to have in pheval.. Do we really need this?

Reply:
Yes, you're right! I can use a built-in module for this